htmd/protocols/Amber_protocols/Production.in

```diff
@@ -0,0 +1,10 @@
+ test namelist
+ &cntrl
+   imin=0, ntx=5, irest=1, 
+   ntc=2, ntf=2, 
+   nstlim=12500000, ntt=3, gamma_ln=1.0,
+   temp0=300.0,
+   ntpr=5000, ntwr=500000, ntwx=5000,
+   dt=0.004, ig=-1, 
+   ntb=2, ntp=1, cut=8.0, barostat=2, ioutfm=1,ntxo=2,
+ /
```

Comment:
Instead of copying this file around, why don't you write it out when using the protocols? Like we do with the ACEMD `input` file.